proposals/description/peg_stability_module.ts

```diff
@@ -0,0 +1,62 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const peg_stability_module: ProposalDescription = {
+  title: 'WETH & DAI Peg Stability Modules',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{wethPSM}'],
+      description: 'Grant Minter Role to (w)ETH PSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{daiPSM}'],
+      description: 'Grant Minter Role to DAI PSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantPCVController(address)',
+      arguments: ['{wethPCVDripController}'],
+      description: 'Give the WETH PCVDripController the PCVController role so that it can withdraw from AAVE'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantPCVController(address)',
+      arguments: ['{daiPCVDripController}'],
+      description: 'Give the DAI PCVDripController the PCVController role so that it can withdraw from Compound'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x1749ca1ca3564d20da6efea465c2a5ae869a9e4b006da7035e688beb14d704e0',
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e'
+      ],
+      description: 'Create PSM_ADMIN_ROLE'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{daiPSM}', '30000000000000000000000000'],
+      description: 'Send 30m DAI to the DAI PSM'
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{wethPSM}', '7500000000000000000000'],
+      description: 'Send 7500 WETH to the WETH PSM'
+    }
```

Comment:
This is a lot of capital to have unutilized at once. Should we consider alternative strategies or leave as-is?

Reply:
For example we could have faster drip rates or use a pull instead of push mechanism

Reply:
These are parameters I discussed with storm. We can jam on this tomorrow and see if we can do less. I wanted each PSM to hold 30m in assets to start, but we can start them off with less and move up.

Reply:
I don't like changing the mechanism by which the PSM receives capital.

I'm up for faster drip rates.